core/slice.go

```diff
@@ -712,11 +712,50 @@ func (sl *Slice) ConstructLocalBlock(header *types.Header) (*types.Block, error)
 	for i, etx := range pendingBlockBody.ExtTransactions {
 		etxs[i] = etx
 	}
-	subManifest := make(types.BlockManifest, len(pendingBlockBody.SubManifest))
+	subBlockHashes := make(types.BlockManifest, len(pendingBlockBody.SubManifest))
```

Comment:
Why did you revert these name changes? They were just changed in the prior commit...

Reply:
Changed it.